PHPhomework2

```diff
@@ -0,0 +1,13 @@
+<?php
+  
+function factorialOfNumber($n)
+{
+  $x = 1;
+  for ($i = 1; $i <= $n; $i++) {
+   $x *= $i;
+  }
+  return $x;
+}
```

Comment:
indent 1 tab
```
function factorialOfNumber($n)
{
    $x = 1;
    for ($i = 1; $i <= $n; $i++) {
        $x *= $i;
    }
    return $x;
}
```

Reply:
Thank you for teaching me that! I'll fix it!